src/containers/Alerts/index.js

```diff
@@ -59,10 +61,4 @@ function AlertsComponent(props) {
 	);
 }
 
-const mapStateToProps = state => {
-	return {
-		alerts: state.alerts.alerts
-	};
-}
-
-export default connect(mapStateToProps)(AlertsComponent);
+export default AlertsComponent;
```

Comment:
@Aringoaway Remove this line and use export default here: https://github.com/TeskaLabs/asab-webui/blob/8580edee187aaa26f64481b63b9cabfac7a5164f/src/containers/Alerts/index.js#L10